docs/user/tutorial/create_page.rst

```diff
@@ -2,41 +2,73 @@
 Create a page
 #############
 
+**********************
+Create your first page
+**********************
 
-Select the *Pages...* menu item from the *Site menu* (*example.com* in the
-example below, though yours may have a different name) in the toolbar.
+django CMS's *Create Page wizard* will open a new dialog box.
 
-.. figure:: /images/pages-menu-item.png
-   :figwidth: 300
-   :align: right
+.. image:: /user/tutorial/images/welcome.png
+   :alt: The 'Create Page wizard'
+   :width: 400
+   :align: center
 
-This reveals the :ref:`side-frame <side-frame>` for page administration.
+Select *Next*, and provide a *Title* and some basic text content for the new page (you'll be
+able to add formatting to this text in a moment), then hit **Create**.
```

Comment:
as a pro tip: you can also double click the first entry and automatically proceed with the selection